.gitignore

```diff
@@ -1,3 +1,4 @@
+.?**.sw*
```

Comment:
What kind of files are being ignored here?

Reply:
it's a poor practice for ignoring the temporary files vim creates
looks like I can just set a tmp directory in vimrc. I'm interested if you have a better solution. Thanks